edwin/client/static/js/components/Timeline.js

```diff
@@ -82,6 +99,22 @@ Timeline.propTypes = {
   params: React.PropTypes.object.isRequired,
 };
 
+class Indicator extends React.Component {
+  render() {
+    let state = this.props.state.toJS().state;
```

Comment:
Uh, nevermind. I just reviewed the code, and Constants worked different than I remembered. Ignore what I said about the just being strings. Sorry.

I think generally the Constants probably shouldn't be considered transparent, and should be treated as opaque objects. That gets complicated when you want to try and display them though.